src/Sylius/Component/Channel/composer.json

```diff
@@ -22,7 +22,7 @@
     "require": {
         "php": ">=5.5.9",
 
-        "doctrine/orm": "2.4.*",
+        "doctrine/orm": "^2.4.8",
```

Comment:
ORM's exceptions should be removed from interface docs. The interface doesn't know anything about the implementation especially about the thrown exceptions.

Let's require `doctrine/common` then.